ui/src/app/components/gke-app-configuration-panels/create-gke-app-button.spec.tsx

```diff
@@ -120,4 +154,169 @@ describe(CreateGkeAppButton.name, () => {
       'You have either run out of initial credits or have an inactive billing account.'
     );
   });
+
+  it('should not allow creating a GKE app with a disk that is too small.', async () => {
+    // Cromwell chosen arbitrarily
+    const tooSmall = appMinDiskSize[AppType.CROMWELL] - 1;
+    await component({
+      createAppRequest: {
+        ...defaultCromwellCreateRequest,
+        persistentDiskRequest: {
+          ...defaultCromwellCreateRequest.persistentDiskRequest,
+          size: tooSmall,
+        },
+      },
+    });
+    const button = await waitFor(() => {
+      const createButton = findCreateButton();
+      expectButtonElementDisabled(createButton);
+      return createButton;
+    });
+
+    await user.pointer([{ pointerName: 'mouse', target: button }]);
```

Comment:
Not necessary by any means, but I think that the `expectTooltip` method in react-test-helpers, might be able to help here.